### PR TITLE
Minor doc fixes.

### PR DESCRIPTION
**Pull request by wjlow**
@benhutchison and I work together at REA Group and found these mistakes in the docs.

### CODE REVIEW

File: org.atnos.site.OutOfTheBox.html [hunk 1]
```diff
@@ -428,7 +428,7 @@ <h3 id="async">Async</h3>
 </code></pre>
 <p>Then we need an interpreter to interpret them as <code class="prettyprint">Futures</code>. The <code class="prettyprint">AsyncFutureInterpreter.create</code> method uses an implicit <code class="prettyprint">ExecutionContext</code> to create the interpreter. If we import the interpreter methods we can finally invoke <code class="prettyprint">runAsyncFuture</code> to get a <code class="prettyprint">Future</code> back.</p>
 <pre><code class="prettyprint">
-val interpreter = AsyncFutures.create
+val interpreter = AsyncFutureInterpreter.create
```
Comment (edmundnoble) [inline org.atnos.site.OutOfTheBox.html]: This actually reflects the current state of the library; the other docs are the ones in need of update (`AsyncTaskInterpreter` is now `AsyncTasks`, etc.).

File: org.atnos.site.OutOfTheBox.html [hunk 2]
```diff
@@ -445,7 +445,7 @@ <h3 id="async">Async</h3>
 
 // the monix service doesn&#39;t require any implicit context!
 val monixService: AsyncTaskInterpreter =
-  org.atnos.eff.monix.addon.AsyncTaskInterpreter</code></pre>
+  org.atnos.eff.addon.monix.AsyncTaskInterpreter</code></pre>
```
Comment (edmundnoble) [inline org.atnos.site.OutOfTheBox.html]: :+1:

File: org.atnos.site.OutOfTheBox.html (hunk 1 above)
Comment (wjlow) [inline org.atnos.site.OutOfTheBox.html]: What import do I need to include to get `AsyncFutures.create` to work?

Comment (edmundnoble) [inline org.atnos.site.OutOfTheBox.html]: When I said "reflects the current state of the library" that was unclear; I meant master at the moment. 2.2.0 does not have these changes.

Comment (wjlow) [inline org.atnos.site.OutOfTheBox.html]: Cool, thanks! Just undid the change.

**Comment by etorreborre** [comment]
Hi @wjlow could you please modify the [spec file](https://github.com/atnos-org/eff/blob/master/jvm/src/test/scala/org/atnos/site/OutOfTheBox.scala#L408) directly? Then I will regenerate the doc online.

**Comment by wjlow** [comment]
Raised https://github.com/atnos-org/eff/pull/53

**Comment by etorreborre** [comment]
closed by #53

**Pull request closed without merging**